core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1214,7 +1214,7 @@ secret_configs:
   to:
   - cluster_groups:
     - non_app_ci
-    name: cluster-secrets-alibaba
+    name: cluster-secrets-alibabacloud
```

Comment:
I will delete `cluster-secrets-alibaba` on clusters after merge.

Reply:
The new one exists
```
oc get secret -n ci cluster-secrets-alibabacloud --context build01
NAME                           TYPE     DATA   AGE
cluster-secrets-alibabacloud   Opaque   4      50m
```

Deleting the old ones:
```
oc delete secret -n ci cluster-secrets-alibaba --context build01 --as system:admin
secret "cluster-secrets-alibaba" deleted

oc delete secret -n ci cluster-secrets-alibaba --context build02 --as system:admin
secret "cluster-secrets-alibaba" deleted

oc delete secret -n ci cluster-secrets-alibaba --context build03
secret "cluster-secrets-alibaba" deleted

oc delete secret -n ci cluster-secrets-alibaba --context build04
secret "cluster-secrets-alibaba" deleted

oc delete secret -n ci cluster-secrets-alibaba --kubeconfig ~/Documents/secret/joe2/sa.config-updater.vsphere.config
secret "cluster-secrets-alibaba" deleted

oc delete secret -n ci cluster-secrets-alibaba --kubeconfig ~/Documents/secret/christian/sa.config-updater.arm01.config
secret "cluster-secrets-alibaba" deleted
```

Reply:
@hongkailiu Big thanks!